src/Application/Interfaces/ISignalRService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using CoduTeam.Application.Messages.EventHadlers;
+using CoduTeam.Application.Messages.Models;
+using CoduTeam.Domain.Entities;
+
+namespace CoduTeam.Application.Interfaces;
+
+public interface ISignalRService
```

Comment:
The point of having it in Application it's to make it package agnostic (but when we have in interface name the actual package name it breaks this rule )